pyproject.toml

```diff
@@ -45,7 +45,7 @@ classifiers = [
 ]
 dependencies = [
     # These are super-old; can/should we update them?
-    "cffi>=1.0.0",
+    "cffi>=1.11",
```

Comment:
Thanks for updating this. Can we also set the min version for cffi in `[build-system.requires]` at the top of the file?